Marlin/ultralcd_implementation_hitachi_HD44780.h

```diff
@@ -624,7 +624,7 @@ static void lcd_implementation_status_screen()
 
 static void lcd_implementation_drawmenu_generic(bool sel, uint8_t row, const char* pstr, char pre_char, char post_char) {
   char c;
-  uint8_t n = LCD_WIDTH - 1 - (LCD_WIDTH < 20 ? 1 : 2);
+  uint8_t n = LCD_WIDTH - 2;
```

Comment:
No.
Since i had my first look on this code i asked me why column 20 is not used and why there is a difference between the 20 column and shorter displays. Yesterday i got it.

Since the first version of this file we have this little error (https://github.com/AnHardt/Marlin/blob/Development/Marlin/ultralcd_implementation_hitachi_HD44780.h#L636) that writes over the first column in the next line of 20 column displays. I suppose -the easy to find fix was to limit the line-length to 19 in all that functions (for symmetry reasons).

@mfeherpataky had a 16 column display at hand and changed the code to use column 16. Because of the different RAM-structure there the first column is not written over. He was the first who differentiated between 20 column and shorter displays.